modules/ai/assets/js/editor/components/prompt-library-link.js

```diff
@@ -0,0 +1,20 @@
+import { Typography, Link } from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+const PromptLibraryLink = ( props ) => {
+	return (
+		<Typography variant="body2" color="text.secondary">
+			{ __( 'For more inspiration, try experimenting with proven prompts from our ' ) }
```

Comment:
Could you please remove the trailing space from the translated text?
```suggestion
			{ __( 'For more inspiration, try experimenting with proven prompts from our' ) } { ' ' }
```